docs/source/fingerprinting.rst

```diff
@@ -1,11 +1,17 @@
-System Fingerprinting
----------------------
-Quipucords is used to inspect and gather information about your IT infrastructure. System information gathered from a single sources or multiple sources are processed to create a summarized set of data called a fingerprint which highlights features that identify they system, such as operating system, architecture, number of CPUs and cores, different products installed and entitlements in use on the system etc. The sections below describe the components of a fingerprint and the process used to create the fingerprint.
+Fingerprints and the Fingerprinting Process
+-------------------------------------------
+The Quipucords scan process is used to discover the systems in your IT infrastructure and to inspect and gather information about the nature and contents of those systems. A *system* is any entity that can be interrogated by the Quipucords inspection tasks through an SSH connection, vCenter Server data, or the Satellite API. Therefore, a system can be a machine, such as a physical or virtual machine, and it can also be a different type of entity, such as a container.
 
+During a scan, information, or a collection of facts, about a system is gathered from a single source (such as a network source only) or multiple sources (such as a network source and satellite source). A *fact* is a single piece of data about a system. Facts are processed to create a summarized set of data for that system that is called a fingerprint. A *fingerprint* is a set of facts that identifies a unique system and the features on that system, such as the operating system, CPU architecture, number of CPUs and cores, the different products that are installed on that system, the entitlements that are in use on that system, and so on.
 
-System Fingerprints
+Fingerprinting is the result of Quipucords analytical processing that occurs when you request a summary report for a scan. If you request a detailed report, you receive the raw facts for that scan without any fingerprinting. When you request a summary report, the deduplication, merging, and post-processing of facts occurs to produce the fingerprint. These processes include identifying installed products and versions from the raw facts, finding consumed entitlements, finding and merging duplicate instances of products from different sources, and finding products installed in nondefault locations, among other steps.
```

Comment:
Fingerprinting actually occurs the moment the scan results are available, not when you request a summary report. The resulting fingerprints are stored in the database and retrieved when a summary report is requested.

Reply:
Not sure? nondefault vs. non-default?

Reply:
RE the non-default vs. nondefault question. RH CCS uses IBM Style as its style guide. IBM style says no hyphens except in some special cases if the dictionary specifies or for clarity--e.g., recreate (play) vs. re-create (create again). So the default is no hyphen.

Reply:
ok 👌

Reply:
RE the comment on fingerprinting and when it occurs: Reworked some of the info in that paragraph per the comments.